### PR TITLE
feat: boj_4949, boj_11399 upload, fix: boj_9012

### PR DESCRIPTION
**Pull request by or-m-or**
feat: boj_4949 upload
feat: boj_11399 upload
fix: boj_9012

### CODE REVIEW

**Review by sjinwoo — APPROVED** [review]
고생해쓰

File: July_2023/src/week27/src/boj_11399_Fri/boj_11399_ormor.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        for(int i=0; i<N; i++){
+            P[i] = Integer.parseInt(st.nextToken());
+        }
+        Arrays.sort(P);
+
+        int sumP = 0;
+        int ansP = 0;
+
+        for(int i=0; i<N; i++){
+            sumP = sumP + P[i];
```
Comment (samyoahri) [inline July_2023/src/week27/src/boj_11399_Fri/boj_11399_ormor.java:28]: sumP += P[i];

File: July_2023/src/week27/src/boj_11399_Fri/boj_11399_ormor.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            ansP = ansP + sumP;
```
Comment (samyoahri) [inline July_2023/src/week27/src/boj_11399_Fri/boj_11399_ormor.java:29]: ansP += sumP;

File: July_2023/src/week27/src/boj_4949_Fri/boj_4949_ormor.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                ans.append(balanceString(line)).append("\n");
+            }
+        }
+        out.write(ans.toString());
+        out.flush();
+        in.close();
+        out.close();
+    }
+
+    private static String balanceString(String line){
```
Comment (samyoahri) [inline July_2023/src/week27/src/boj_4949_Fri/boj_4949_ormor.java:35]: 굿굿

**Review by samyoahri — APPROVED** [review]

**Pull request merged**